classes/DirtEvent.sc

```diff
@@ -102,24 +119,38 @@ DirtEvent {
 		};
 
 		sustain = ~sustain.value;
+	
 		sustain = sustain ?? {
```

Comment:
The changes below: are you sure strudel couldn't translate its logic to the way tidalcycles works? It will be very confusing to change all that …